massa-bootstrap/src/server.rs

```diff
@@ -35,14 +33,22 @@ use crate::{
 /// handle on the bootstrap server
 pub struct BootstrapManager {
     join_handle: JoinHandle<Result<(), BootstrapError>>,
-    manager_tx: mpsc::Sender<()>,
+    manager_tx: mpsc::Sender<BSInternalMessage>,
+}
+
+#[derive(Debug)]
+enum BSInternalMessage {
+    // Signals the Manager to stop
+    Stop,
+    // Client has completed a bootstrap
+    Complete,
```

Comment:
I don't think we need this enum for the time-being. The only use for this seems to be for running a massa_trace. This could be integrated into a dropping of a `Arc<()>` wrapper. Would be ideal to not use the Arc-pattern though.

Reply:
Yes I don't think it's useful if we have the arc counter

Reply:
I'll revert. Probably only still in there due to sentimental attachment for my more ambitious attempt that a backtracked from :(

Reply:
reversion done.

Reply:
@AurelienFT  do you feel this is resolved?

Reply:
Yes